src/Main.js

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import Error from "./Error";
+import Resturants from "./Resturants";
+import { Button, Container, Form } from "react-bootstrap";
+import axios from "axios";
+
+class Main extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      displayInfo: false,
+      city: '',
+      cityData: {},
+      restresturantData: [],
+      locationData: [],
+      weatherData: [],
+      errorData: false
+    }
+  }
+
+  handleSearchInput = e => {
+    let cityName = e.target.value;
+    this.setState({
+      city: cityName
+    },
+      () => console.log(this.state.city)
+    )
+  }
+
+  displaySearch = async (e) => {
+    e.preventDefault();
+    try {
+      let url = `https://us1.locationiq.com/v1/search?key=${process.env.REACT_APP_LOCATIONIQ_KEY}&q=${this.state.city}&format=json`;
+
+      
+
+      // let url2 = `https://maps.locationiq.com/v3/staticmap?key=${process.env.REACT_APP_LOCATIONIQ_KEY}&center=${this.state.cityData.lat},${this.state.cityData.lon}&zoom=9`;
```

Comment:
Make this the definition for a state variable to pass down into Maps as props. Also, create a variable that doesn't use state for the lat and lon values.